### PR TITLE
add empty alt attribute to the image component

### PR DESCRIPTION
**Pull request by poly4concept**

### CODE REVIEW

**Comment by FlyteWizard** [comment]
@poly4concept Hey! In the future if you add `closes #180` in the PR, it will automatically close the issue once the PR is merged. A cool little fact about GitHub workflow! 😄

**Comment by poly4concept** [comment]
Oh @FlyteWizard  thank you for that suggestion 
I'm pretty new to open source and I will be glad to be among a community where I would learn or these workflow tips and more.
Once again, thank you

File: src/app/landing/index.tsx [hunk 1]
```diff
@@ -44,10 +44,12 @@ export default function Landing() {
       <GridItem display={{ base: 'none', xl: 'initial' }} colSpan={1}>
         <Container>
           <Image
+            alt=""
             src={process.env.PUBLIC_URL + '/assets/computer.svg'}
             sx={{
               filter: 'drop-shadow( 1.5rem 1rem 1.75rem rgba(0, 0, 0, .25) )',
             }}
+
```
Comment (keithradford) [inline src/app/landing/index.tsx:52]: There is a linting error here - there shouldn't be an empty line.
If you run `npm run lint:fix` it should fix it 😄

**Comment by keithradford** [comment]
@poly4concept Hey! Just checking in to see the status of this PR, is this something you are still working on?

**Comment by poly4concept** [comment]
Hi @keithradford, yeah I'm still working on it.
I will make a pull request today

**Comment by FlyteWizard** [comment]
Hey! Unfortunately, the repository has been refactor since you created your PR. I've added the correct page in a code snippet below. 

Feel free to close this PR, update your Fork, and create a new branch with the required changes plus create a new PR. (Or you can update your current branch from main and re-apply the changes and leave this PR open). 

https://github.com/VikeLabs/courseup/blob/f711c46ed245d2ad8d5ce7e7c8efac7d8cd835c8/src/pages/home/containers/Landing.tsx#L47-L52

Let us know if you need any help. 😄

**Comment by aomi** [comment]
@poly4concept I'm going to close and unassign you from this ticket as it's been stale for a while now. You're free to reopen this PR if the issue hasn't been addressed by then!

**Pull request closed without merging**